setup.py

```diff
@@ -36,7 +36,7 @@ def load_requirements(*requirements_paths):
 
 setup(
     name='ora2',
-    version='3.8.2',
+    version='4.0.0',
```

Comment:
This should be 3.9.0. Why are we adding a major version bump here

Reply:
we are dropping support for django22

Reply:
We are following this approach to update minor version for any version drops and we will be releasing major version when we move to next major version of Django.

Reply:
we are already running on Django 3.2 and only dropping the testing environment for Django 2.2